tessera.lic

```diff
@@ -0,0 +1,74 @@
+=begin
+  Documentation:  Tessera script based upon concept around Almanac.lic.
+=end
+
+custom_require.call(%w[common common-items drinfomon])
+
+class Tessera
+
+  def initialize
+    settings = get_settings
+    tessera_retry_interval = settings.tessera_retry_interval
```

Comment:
Related to the other comment, I suggest making this an instance var (`@tessera_retry_interval`).